CMakeLists.txt

```diff
@@ -473,8 +475,12 @@ add_custom_target(metadata_airframes
 	USES_TERMINAL
 )
 
+file(GLOB_RECURSE yaml_config_files ${PX4_SOURCE_DIR}/src/modules/*.yaml
```

Comment:
It's worth getting rid of all cmake globbing to avoid possible surprises with incremental builds.

Reply:
Absolutely. This was just the quickest way to achieve it. Essentially I want to tell cmake 'give me all module configs for all possible modules'. And adding a new config with all modules does not seem right for that.